test/clj/rems/test_browser.clj

```diff
@@ -967,9 +988,30 @@
         (btu/fill-human :fields-0-title-sv "Description (SV)")
 
         (btu/scroll-and-click :save)
-        (btu/wait-visible {:tag :h1 :fn/text "Form"})
         (btu/wait-page-loaded)
-        (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))))
+        (btu/wait-visible {:tag :h1 :fn/has-text "Form"})
+        (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"})))
+
+      (testing "check that error message is present on field empty"
+        (btu/scroll-and-click {:fn/has-class :edit-form})
+        (btu/wait-page-loaded)
+        (btu/wait-visible {:tag :h1 :fn/text "Edit form"})
+
+        (btu/scroll-and-click :fields-0-type-description)
+        (btu/fill-human :fields-0-info-text-en "Info text (EN)")
+        (btu/fill-human :fields-0-info-text-fi "Info text (FI)")
+        (btu/fill-human :fields-0-info-text-sv " ")
+
+        (btu/scroll-and-click :save)
+
+        (btu/visible? {:tag :textarea :id :fields-0-info-text-sv :fn/has-class :is-invalid})
+        (btu/visible? {:tag :div :class :invalid-feedback :fn/has-text "Field \"Field description (optional)\" is required."})
```

Comment:
hmm does it not show which language is missing? oh well, that can be fixed later/separately.